src/codegen.c

```diff
@@ -130,6 +130,7 @@ int get_code_length(ir_instr_t *ii)
     case OP_add:
     case OP_sub:
     case OP_mul:
+    case OP_sdiv:
```

Comment:
Any strong reason to specify "sdiv" instead of "div" in general? We would like to support RISC-V later.

Reply:
I suppose udiv will be imported later. If not possible, I can change it!

Reply:
> Any strong reason to specify "sdiv" instead of "div" in general? We would like to support RISC-V later.

Hi @jserv 

any plan/roadmap to support RISC-V?
I am planning to use shecc in my compiler (online) class and just start to replace the arm backend to RISC-V.
It would be a good news if you are implementing RISC-V backend.
(drive by comment. sorry for thread hijacking)